src/content/development/contribution-guide/_index.en.md

```diff
@@ -46,7 +46,12 @@ To be expanded
 
 ## Release Cycles
 
-To be expanded
+The Submariner project follows time based release cycles, each cycle is 16 weeks long.
```

Comment:
```suggestion
Submariner follows 16 week long time-based release cycles.
```

Reply:
I think original is easier to understand, with this one it isnt clear what time-based relates to.
We want to emphasize that the cycles are time based hence I wrote it explicitly.